salt/states/archive.py

```diff
@@ -1076,6 +1083,11 @@ def extracted(name,
                           ))
         return ret
 
+    if clean and clean_parent:
+        ret['comment'] = "You can't set both 'clean' and 'clean_parent' to True."
```

Comment:
We don't typically use the phrasing "you can't". Instead, can you change this to `Only one of 'clean' and 'clean_parent' can be set to True`?

Reply:
thanks, done!